scripts/globals/status.lua

```diff
@@ -2587,6 +2587,7 @@ xi.mobMod =
     TARGET_DISTANCE_OFFSET = 73, -- Adjusts how close a mob will move to it's target. 12 = 1.2 yalm. Positive values to go closer, negative farther.
     ONE_WAY_LINKING        = 74, -- Will link with other mobs in its party (typically the same mob family) while roaming, but will not let others link with it once engaged
     CAN_PARRY              = 75, -- Check if a mob is allowed to have parry rank (Rank Value 1-5)
+    MOBMOD_NO_WIDESCAN     = 76, -- Disables widescan for a specific mod
```

Comment:
I would change the name to NO_WIDESCAN to match the rest of the mobmods

Reply:
Agreed on the above comment.  We know this is a mobmod due to the enum declaration.

Reply:
Very well. Thank you.